From e0a73cec209e0168a7c455792394c2e2a978db73 Mon Sep 17 00:00:00 2001
From: Alexandru Costache <alexandru@balena.io>
Date: Tue, 5 Mar 2019 14:05:52 +0100
Subject: [PATCH] Integrate machine independent resin environment configuration

This patch integrates resin default environment
configuration in an u-boot which has a version
with Kconfig support.

This is a rebase of the original version
of the patch done by Andrei Gherzan.

Signed-off-by: Alexandru Costache <alexandru@resin.io>

Upstream-Status: Inappropriate [configuration]
---
 include/env_default.h | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/include/env_default.h b/include/env_default.h
index 54d8124..5632ec9 100644
--- a/include/env_default.h
+++ b/include/env_default.h
@@ -9,6 +9,8 @@
 
 #include <env_callback.h>
 
+#include <env_resin.h>
+
 #ifdef DEFAULT_ENV_INSTANCE_EMBEDDED
 env_t environment __UBOOT_ENV_SECTION__(environment) = {
 	ENV_CRC,	/* CRC Sum */
@@ -21,6 +23,7 @@ static char default_environment[] = {
 #else
 const uchar default_environment[] = {
 #endif
+	RESIN_ENV
 #ifndef CONFIG_USE_DEFAULT_ENV_FILE
 #ifdef	CONFIG_ENV_CALLBACK_LIST_DEFAULT
 	ENV_CALLBACK_VAR "=" CONFIG_ENV_CALLBACK_LIST_DEFAULT "\0"
-- 
2.7.4

